apps/meteor/app/api/server/api.js

```diff
@@ -311,21 +330,47 @@ export class APIClass extends Restivus {
 		return routeActions.map((action) => this.getFullRouteName(route, action, apiVersion));
 	}
 
+	checkPermissions(options) {
```

Comment:
(pick alert 2)

this method could be a function in file's scope right?

I'd also recommend not changing the `options` param, but instead having  the `permissionRequired` as the return value.

having this function in TS and having a unit test would also be good